src/gui-wizard-gtk/wizard.c

```diff
@@ -1362,8 +1362,7 @@ static void terminate_event_chain(int flags)
     if ((flags & TERMINATE_WITH_RERUN))
         return;
 
-    free(g_event_selected);
-    g_event_selected = NULL;
+    g_clear_pointer(&g_event_selected, g_free);
```

Comment:
The second argument should be `free` if `g_event_selected` was allocated with `malloc`. (Same with the others.)

Reply:
Fixed in the cases of `g_event_selected`.  
`warning` at src/gui-wizard-gtk/wizard.c:2339 comes from `g_strdup_printf`, `text_copy` in src/lib/event_xml_parser.c comes from `g_strndup` and `linebuf` at src/lib/stdio_helpers.c:60 comes from `g_realloc`, so those should be fine with `g_free`, shouldn't they?

Reply:
Yes, that seems correct.